### DISCUSSION TITLE
Way to cancel committing while editing message?

### DISCUSSION THREAD

**Discussion by xiaochuanyu** [NeogitOrg/neogit#403] [Q&A] [answered]
Is there a way to cancel committing while editing message?

**Comment by alexmozaidze** [comment, marked as answer]
You leave the message empty and quit the buffer (`:q!`)

**Reply by ten3roberts** [reply]
In general, the commit will be aborted if the commit message is empty (save for comments), so any keybind which quits the buffer works. If you have `set confirm` you can just make neovim discard the buffer by closing it through any keybind.
